<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TuringEmulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Условие задачи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_4"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Исходный алфавит</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Дополнительный алфавит</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Комментарии</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Лента</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Выполнение</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Скорость</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="actionQT"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Восстановить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Запуск</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Следующий шаг</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Остановить</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Предыдущий шаг</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Автор</string>
   </property>
  </action>
  <action name="actionQT">
   <property name="text">
    <string>QT</string>
   </property>
  </action>
  <action name="action_15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Очень быстро</string>
   </property>
  </action>
  <action name="action_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Быстро</string>
   </property>
  </action>
  <action name="action_17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Средне</string>
   </property>
  </action>
  <action name="action_18">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Медленно</string>
   </property>
  </action>
  <action name="action_19">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Очень медленно</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Задать строку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
